cas-server-webapp/src/main/webapp/css/cas.css

```diff
@@ -454,6 +454,10 @@ footer a:link, footer a:visited {
     width: 100%;
 }
 
+#inst-login .need-help {
+    font-size: 15px;
+}
```

Comment:
Slightly increased the font size for "Not your institution?" as a response to demo changes

![inst-login-auto-select](https://user-images.githubusercontent.com/3750414/79933467-184ff500-841e-11ea-934a-4d392a0ca649.png)